pkg/users/account-create-dialog.js

```diff
@@ -118,6 +160,10 @@
         return _("The full name must not contain colons.");
 }
 
+function validate_home_dir(dir) {
+    return cockpit.spawn(["test", "!", "-d", dir], { superuser: "require" });
+}
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18234-20230202-125519-871b841d-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)